language/Dutch/strings.po

```diff
@@ -1255,7 +1255,7 @@ msgstr "Normale tekst"
 
 #: src/frontend/mame/ui/custui.cpp:452
 msgid "Selected color"
-msgstr "Geselecteerde kleur"
+msgstr "Geselecteerd kleur"
```

Comment:
was correct, kleur is een "de" woord

Reply:
This string is used in the ui color menu. It means the color for "selected" items in the ui. So "geselecteerde kleur" did not make sense. Do you have a better translation?

Reply:
Ok understood, it's fine then